ctb_metadata_files/metadata.graphql

```diff
@@ -33,6 +33,7 @@ type ONS_Variable {
   Variable_Type: Variable_Type!
   Quality_Statement_Text: String
   Quality_Summary_URL: String
+  Geography_Hierarchy_Order: String
```

Comment:
Should this be non null?

Reply:
No, it will be null for non-geographic variables.

Reply:
Yes, realised that after a few minutes.